src/GoogleBooks.php

```diff
@@ -70,7 +70,7 @@ public function __construct($options = [])
         $this->maxResults = isset($options['maxResults']) ? $options['maxResults'] : null;
     }
 
-    protected function raw($endpoint, $params = [], $method='GET')
+    public function raw($endpoint, $params = [], $method='GET')
```

Comment:
Was this just to make phpspec happy? If so, is there another way?

Reply:
no, this is so I could keep the API calls inside the `GoogleBooks` class and access them through the `fetch()` method inside the `LibraryBuilder` class.

`LibraryBuilder` class takes a `GoogleBooks` dependency (client) and then can access anything it needs from the `LibraryBuilder`.

This way we keep communication with GoogleBooks inside the GoogleBooks class and the data manipulation methods inside the `LibraryBuilder` class

Reply:
Ah, yeah, didn't see that. Makes sense.